internal/runtimehandlerhooks/utils_test.go

```diff
@@ -48,6 +48,14 @@ var _ = Describe("Utils", func() {
 				input:    Input{cpus: "4-13", mask: "ffff,ffffc00f", set: true},
 				expected: Expected{mask: "0000ffff,ffffffff", invMask: "00000000,00000000"},
 			}),
+			Entry("clear a single bit that was one when odd mask is present", TestData{
+				input:    Input{cpus: "9", mask: "fff", set: false},
+				expected: Expected{mask: "00000dff", invMask: "0000f200"},
+			}),
+			Entry("clear two bits from a short mask", TestData{
+				input:    Input{cpus: "2-3", mask: "ffffff", set: false},
+				expected: Expected{mask: "00fffff3", invMask: "0000000c"},
+			}),
```

Comment:
do we have a test that checks the mask is not bigger than it is supposed to be (per the [very nice!]) comment you added in utils.go

Reply:
This very test (short mask) should cover that case I hope. I am not running an exhaustive check though..

Reply:
should be a good enough start